src/api/url.js

```diff
@@ -0,0 +1,20 @@
+import CartoValidationError from './error-handling/carto-validation-error';
```

Comment:
@davidmanzanares @jgoizueta if you see that this refactor is no necessary I can revert it

Reply:
Please keep it 😄